entrypoint

```diff
@@ -24,6 +24,11 @@ if [ -n "$INPUT_CTESTFLAGS" ]; then
     ctest+=( "$INPUT_CTESTFLAGS" )
 fi
 
+cmake=( cmake )
+if [ -n "$INPUT_CMAKEFLAGS" ]; then
+    cmake+=( "$INPUT_CMAKEFLAGS" )
+fi
+
```

Comment:
I've been planning on doing this, thanks!
Have you checked if it works with multiple flags?

Reply:
:thinking: Looks like it wasn't, the quotes around `INPUT_CMAKEFLAGS` were getting propagated to the cmake command and that was causing it to ignore all of them after the first one.  I pushed out a new one that fixes this by removing the quotes.

[I tested it out here on one of my repos, and it's working toggling both options in the same CMakeLists.txt](https://github.com/washingtondc-emu/washingtondc/blob/github_actions_invariants/.github/workflows/build_linux.yml).